lib/livebook_cli/server.ex

```diff
@@ -19,6 +19,9 @@ defmodule LivebookCLI.Server do
 
     Available options:
 
+      --autosave-dir       The directory where notebooks with no file are persisted.
```

Comment:
@josevalim any other naming ideas welcome :D

Reply:
I changed `-dir` to `-path` to stay in line with `--root-path`, but autosave is still in question.

Reply:
I like autosave.